sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -28,7 +28,7 @@ export class CreateInstitutionDto {
   website: string;
   @IsOptional()
   regulatingBody: string;
-  @IsNotEmpty()
+  @IsDate()
```

Comment:
are you sure if `@IsDate()` works? I think formio returns the date string

Reply:
Yes @IsDate works well with date string.

Reply:
Here is an example of same scenario with @IsDate() class validator. 

![image](https://user-images.githubusercontent.com/54600590/164542741-b5ce583d-f6df-4ac3-98b5-a66e5b3d95ca.png)

![image](https://user-images.githubusercontent.com/54600590/164542637-1fca1743-690c-4407-96eb-c908be14da57.png)

Reply:
Thanks @dheepak-aot for the details